### ISSUE TITLE
DICe doesn't create a full field of coordinates within the picture

### ISSUE THREAD

**Issue by Tristanrommers** [dicengine/dice#288]
I am currently working on a DIC project where I want to obtain a stress strain curve of my testpieces. While I am trying to create a full field of coordinates on the pictures of my testpiece, DICe stops somewhere around halfway. Does anyone know what might be the problem in this case? I already tried changing the ISO value of the pictures, that doesn't seem to be the problem. A better pattern could be a solution but then again, DICe still managed to create a field in the first half. Is there anyone who had the same issue or knows a solution? 

![question](https://user-images.githubusercontent.com/130751815/232015970-cef1f2a6-d996-42d6-b75f-f9e5ee93ae0e.JPG)

**Comment by dicengine** [comment]
Did you get any error messages in the console window? One thing to try would be to set the SSSIG threshold to 0 and see if that fixes it.

**Comment by Tristanrommers** [comment]
Thank you for your reply. Setting the SSSIG threshold to 0, results in a larger area of coordinates. However there is still a region without coordinates. Unfortunately the program doesn't provide an error code. When I reduced the region of interest to the part where there aren't any coordinates, I did receive the following error message:

Throw number = 1

Throw test that evaluated to true: comm_->get_rank()==0&&subset_centroids_x.size()<=0
